README.md

```diff
@@ -44,6 +50,7 @@ If you want to access other content types, you can easily have us grab them by c
 
 ```js
 new Rooftop({
+  addDataTo: locals,
```

Comment:
i'd really like to define the `locals` object in all of these snippets so we avoid confusion.